src/services/utilities.ts

```diff
@@ -1922,10 +1922,6 @@ namespace ts {
         }
     }
 
-    export function isMemberSymbolInBaseType(memberSymbol: Symbol, checker: TypeChecker): boolean {

```

Comment:
I inlined this because it's only used once and I had to change that line of code anyway